src/ChunkMap.h

```diff
@@ -246,6 +248,10 @@ class cChunkMap
 	Returns true if all block entities processed, false if the callback aborted by returning true. */
 	bool ForEachBlockEntityInChunk(int a_ChunkX, int a_ChunkZ, cBlockEntityCallback & a_Callback);  // Lua-accessible
 
+	/** Calls the callback for brewingstand in the specified chunk.
+	Returns true if all brewingstands processed, false if the callback aborted by returning true. */
+	bool ForEachBrewingstandInChunk(int a_ChunkX, int a_ChunkZ, cBrewingstandCallback & a_Callback);  // Lua-accessible
```

Comment:
Oh. Why not have authors check for themselves the block entity they want in the generic function? Its a lot of effort to remember/implement new templates for each new block entity.